app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -71,60 +39,68 @@ To be eligible for a bursary, you’ll need at least one of the following:
 
 If your course combines 2 subjects (for example, English and history), at least 50% of your course content must be made up of the bursary subject to be eligible.
 
-You’ll also need to meet the bursary scheme’s individual terms and conditions.
+#### How bursaries are paid
 
-[Find out more about your eligibility for teaching bursaries](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2024-to-2025).
+A bursary is paid over a minimum of 10 months by your teacher training provider. They will confirm when you will get the payments. 
 
-### Scholarships
+You will not need to pay tax or National Insurance on bursary payments. 
 
-#### Eligibility
+#### How to get a bursary
+
+You do not need to apply for a bursary. When you apply for teacher training, your provider will check if you're eligible for a bursary. If you are, your provider will confirm this in writing before you start your course. 
+
+## Eligibility for scholarships
```

Comment:
Currently we say `You must be eligible for student finance to receive a bursary or scholarship (unless you’re training to teach physics or languages).`

Is this no longer the case?